Hello/vcpkg-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+    "registries": [
+      {
+        "kind": "artifact",
+        "location": "https://aka.ms/vcpkg-ce-default",
+        "name": "microsoft"
+      },
+      {
+        "kind": "artifact",
+        "location": "https://artifacts.keil.arm.com/vcpkg-ce-registry/registry.zip",
+        "name": "arm"
+      }
+    ],
+    "requires": {
+      "microsoft:tools/kitware/cmake": "^3.25.2",
+      "microsoft:ninja": "^1.10.2",
+      "arm:compilers/arm/armclang":"^6.20.0",
+      "arm:compilers/arm/arm-none-eabi-gcc": "^13.2.1",
+      "arm:compilers/arm/llvm-embedded": "^17.0.1-0",
+      "arm:tools/open-cmsis-pack/cmsis-toolbox": "^2.1.0-0",
+      "arm:models/arm/avh-fvp": "^11.22.39",
+      "arm:debuggers/arm/armdbg": "^6.0.0"
```

Comment:
Is the arm debugger strictly required? The Arm Debugger extension will ensure that the armdbg artifact is available in the VS Code environment.

Reply:
Didn't know; it was not automatically installed last time I debugged examples.

Reply:
Did you have the Extension Arm Debugger Preview v 0.3.0 installed?

Reply:
No, I have Arm Embedded Debugger 1.0.13 installed.